modules/analysis/src/service/render.rs

```diff
@@ -1,7 +1,7 @@
 use super::*;
 
 impl AnalysisService {
-    pub fn render_dot(&self, sbom: &str) -> Option<String> {
+    pub fn render_dot(&self, graph: Arc<PackageGraph>) -> Option<String> {
```

Comment:
Is the `Arc` really required? Or is a `&PackageGraph` good enough?

Reply:
ref is good enough.